Formula/ciphey.rb

```diff
@@ -0,0 +1,187 @@
+class Ciphey < Formula
+  include Language::Python::Virtualenv
+
+  desc "Automatic decryption and decoding tool"
+  homepage "https://github.com/Ciphey/Ciphey"
+  url "https://files.pythonhosted.org/packages/a5/db/9e0411803c768cd7f5c6986c9da406ae7e4b6b6a1d8ad0dc191cff6dbdaf/ciphey-5.14.0.tar.gz"
+  sha256 "302a90261e9acc9b56ea29c313192f0c6f6ce112d37f4f9d404915052e19bf09"
+  license "MIT"
+
+  depends_on "boost" => :build
+  depends_on "cmake" => :build
+  depends_on "poetry" => :build
+  depends_on "swig" => :build
+  depends_on "libyaml"
```

Comment:
Just curious, why is libyaml required?

Reply:
We build the `resource`s from source and if I'm not wrong PyYAML requires `libyaml` to be installed, for it to be built from source.

Reply:
Ah ok that would make sense. We had just set it as a dependency in the pyyaml port, which ciphey would itself then depend on.

Thanks @nandahkrishna for the explanation :)